ui/src/components/HelpAccordian.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react";
+import styled from "styled-components";
+import { colors } from "../styles/colors";
+import Accordion from "@material-ui/core/Accordion";
+import AccordionSummary from "@material-ui/core/AccordionSummary";
+import AccordionDetails from "@material-ui/core/AccordionDetails";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { Body1 } from "../styles/typography";
+
+const StyledAccordianSummary = styled(AccordionSummary)`
+  flex-direction: row-reverse;
+`;
+
+const StyledBox = styled.div`
+  background: ${colors.neutral};
+  border-radius: 10px;
+  padding: 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledAccordian = styled(Accordion)`
+  && {
+    box-shadow: none;
+  }
+
+  .MuiAccordionSummary-root {
+    padding: 0px;
+  }
+
+  .MuiAccordionSummary-content {
+    margin: 10px 0px 10px 20px;
+  }
+
+  .MuiAccordionSummary-content.Mui-expanded {
+    margin: 10px 0px 10px 20px;
+    color: ${colors.primaryDark1};
```

Comment:
It's a small thing but our primaryDark1 is the wrong dark teal, it should be `#2D6394` according to Figma but right now it's `#2D8694` which is leading to a discrepancy
![image](https://user-images.githubusercontent.com/19617248/101397631-f3b86880-389a-11eb-98bb-b040f1731708.png)
vs.
![image](https://user-images.githubusercontent.com/19617248/101397655-fca93a00-389a-11eb-87b8-2091cf4e4397.png)

Reply:
Yeah, I was wondering this as well. The Figma colour styles are #2D6394 but on the design system page it says #2D8694.